classes/image_types_ostree.bbclass

```diff
@@ -37,13 +39,15 @@ IMAGE_CMD:ostree () {
     # Copy required as we change permissions on some files.
     tar --xattrs --xattrs-include='*' -cf - -S -C ${IMAGE_ROOTFS} -p . | tar --xattrs --xattrs-include='*' -xf - -C ${OSTREE_ROOTFS}
 
-    for d in var/*; do
-        if [ "${d}" != "var/local" ]; then
-            if [ -d ${d} ] && [ ! -L ${d} ]; then
-                ostree_rmdir_helper ${d}
-            fi
-        fi
-    done
+    # Just preserve var/local
+    if [ -d var/local ]; then
+        mv var/local var-local
+    fi
+    ostree_rmdir_helper var
+    mkdir var
+    if [ -d var/local ]; then
```

Comment:
I know this was merged quite a while ago but I wanted to ask: shouldn't this `if` here be checking directory `var-local` (rater than `var/local`)?

Reply:
yes, you are right. good catch.
can you propose a fix for this? otherwise I can prepare one tomorrow.

Reply:
Thanks for the answer. Sure, I'll make the fix then.

Reply:
@quaresmajose I've created the PRs fixing this on kirkstone (that's what we need at Toradex) and master. Should I do it on some other branch?

Reply:
That's fine by me, the others are already stale. Thanks